web/src/engine/websites/NetTruyenOK_e2e.ts

```diff
@@ -20,7 +20,5 @@ const config: Config = {
         size: 193_514,
```

Comment:
May use one of the last images in chapter to ensure there is no lazy loading

Reply:
Already merged but ok

Reply:
no problem